src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_policies.erl

```diff
@@ -67,7 +67,35 @@ malformed_request_message(Any, _Req, _State) ->
     error({unexpected_malformed_request_message, Any}).
 
 to_json(Req, #base_state{chef_db_context = DbContext,
-                         resource_state = StubRec
+                         organization_guid = OrgId,
+                         reqid = ReqId
                         } = State) ->
-    Names = chef_db:list(StubRec, DbContext),
-    {chef_json:encode(lists:sort([ hd(NameAsList) ||NameAsList <- Names])), Req, State}.
+    case chef_db:list_all_policy_revisions_by_orgid(DbContext, OrgId) of
+        {error, Why} ->
+            Report = {list_all_policy_revisions_by_orgid, {Why, ReqId}},
+            lager:error("~p", [Report]),
+            error(Report);
+        AllRevisions ->
+            BaseEJSON = build_base_policy_list_ejson(Req, AllRevisions),
+            io:format("BASE EJSON:~p~n", [BaseEJSON]),
+            PolicyEJSON = build_policy_list_ejson(AllRevisions, BaseEJSON),
+            io:format("ALL REVS:~p~n", [PolicyEJSON]),
```

Comment:
derp, I need to put TODOs next to these since I grep for that before making pull requests.